update_lunch.py

```diff
@@ -0,0 +1,123 @@
+import urllib2
+import urllib
+import string
+import string
+import logging
+import datetime
+import re
+from google.appengine.ext import ndb
+from HTMLParser import HTMLParser
+
+class MLStripper(HTMLParser):
+    def __init__(self):
+        self.reset()
+        self.fed = []
+    def handle_data(self, d):
+        self.fed.append(d)
+    def get_data(self):
+        return ''.join(self.fed)
+
+class Lunch(ndb.Model):
+    summary = ndb.StringProperty(required=True)
+    # description is a list of lines in the description
+    description = ndb.StringProperty(repeated=True)
+    day = ndb.DateProperty(required=True)
+
+def strip_tags(html):
```

Comment:
is strip_tags or MLStripper used? Looks like you have your own tag stripper already.